axon-framework/sagas/implementation.md

```diff
@@ -129,3 +102,103 @@ To configure caching, simply wrap any `SagaStore` in a `CachingSagaStore`. The c
 
 The latter two arguments may refer to the same cache, or to different ones. This depends on the eviction requirements of your specific application.
 
+## Configuring a Saga
+
+Although a Saga requires a [manager](#saga-manager), [repository / store](#saga-repository-and-saga-store) and wiring to the right message busses, configuring a Saga is straightforward.
+When using the Configuration API, Axon will use sensible defaults for most components.
+
+As a specific type of [Event Handling Component](../events/event-handlers.md), configuration of a Saga is closely related to the configuration of [Event Processors](../events/event-processors/README.md).
+Due to this, configuring a processor will impact the behaviour of a Saga, albeit on a non-functional level.
+The configuration of [error handling](../events/event-processors/README.md#error-handling) or [processor assignment rules](../events/event-processors/README.md#assigning-handlers-to-processors), for example, are thus equally valid for Sagas as long as the right processor name is used during configuration.
+
+> **Default Saga Processor name**
+> 
+> As a Saga is a type of event handler, it is part of an Event Processor.
+> Without defining any [assignment rules](../events/event-processors/README.md#assigning-handlers-to-processors), a Saga's processor name equals the Saga name appended with "Processor",
+>
+> With a Saga called `MySaga`, that would mean the processor is called `MySagaProcessor`.
+
+Internally, Axon uses a `SagaConfigurer` to construct the Saga, Saga Manager, Saga Repository and Saga Store.
+A default configuration for a Saga called `MySaga` would look as follows:
+
+{% tabs %}
+{% tab title="Axon Configuration API" %}
+As a specific type of event handler, registering a Saga is done through the `EventProcessingConfigurer`:
+
+```java
+class AxonConfig {
+
+    void configureMySaga(EventProcessingConfigurer eventProcessingConfigurer) {
+        eventProcessingConfigurer.registerSaga(MySaga.class);
+    }
+}
+```
+{% endtab %}
+
+{% tab title="Spring Boot AutoConfiguration" %}
+In a Spring environment, the Saga implementation should be annotated with `@Saga` to auto-configure it:
+
+```java
+import org.axonframework.spring.stereotype.Saga;
+
+@Saga
+class MySaga {
+    // saga implementation left out...
+}
+```
+{% endtab %}
+{% endtabs %}
+
+Although the defaults lead us to a working Saga environment, it is recommended to define the [`SagaStore`](#saga-repository-and-saga-store) to use.
+The `SagaStore` represents the mechanism that 'physically' stores the Saga instances, for which it uses the `AnnotatedSagaRepository` \(the default\) to store and retrieve Saga instances.
+Providing no storage means for a Saga means Axon defaults to an `InMemorySagaStore`, thus not persisting the Saga on shutdown.
```

Comment:
Maybe change 
Providing no storage means for a Saga means Axon defaults to...
to...
If no `SagaStore` is configured, Axon defaults to....